### PR TITLE
Dash/replace csv with jsonconnector

### PR DESCRIPTION
**Pull request by pawellysy**
Replaced `CSVconnector` with `JSONConnector` in the demos visible on main page.

### CODE REVIEW

**Review by karolkolodziej — APPROVED** [review]
Looks good!

**Review by goransle — APPROVED** [review]

**Comment by sebastianbochan** [comment]
Can we change it in cypress test as well?

File: samples/dashboards/demo/datagrid-sync/demo.js [hunk 1]
```diff
@@ -38,37 +45,47 @@ const chartOptions = {
...
+                            },
+                            {
+                                id: 'dashboard-col-2'
+                            }
+                        ]
+                    }
+                ]
+            }
+        ]
     },
```
Comment (Vazonik) [inline samples/dashboards/demo/datagrid-sync/demo.js:89]: This results in such long, in my opinion, unnecessary lines of code.

File: samples/dashboards/demo/datagrid-sync/demo.js [hunk 2]
```diff
@@ -108,7 +126,8 @@ Dashboards.board('container', {
                     text: 'Dragging points <em>will not update</em> the grid'
                 }
             })
-        }, {
+        },
+        {
```
Comment (Vazonik) [inline samples/dashboards/demo/datagrid-sync/demo.js:130]: Similarly in these cases

**Review by Vazonik — APPROVED** [review]
Just a small note, but was breaking the blocks of brackets/braces on purpose? I think it would be better to restore it how it was before, i.e.:
```js
connectors: [{
    ...
}]
```
instead of 
```js
connectors: [
    {
        ...
    }
]
```

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**